legend-engine-language-pure-dsl-service/src/main/java/org/finos/legend/engine/language/pure/dsl/service/grammar/to/ServiceGrammarComposerExtension.java

```diff
@@ -97,6 +97,13 @@ private static String renderService(Service service, PureGrammarComposerContext
             serviceBuilder.append(getTabString()).append("test: ");
             serviceBuilder.append(HelperServiceGrammarComposer.renderServiceTest(service.test, context));
         }
+        if(service.tags != null)
```

Comment:
need a check for ListIterate.any(service.tags, t -> "metricGroup".equals(t.key)). Similarly only pass tags with key as "metricGroup" to renderServiceTelemetry